core/src/test/scala/org/apache/spark/InternalAccumulatorSuite.scala

```diff
@@ -188,13 +188,18 @@ class InternalAccumulatorSuite extends SparkFunSuite with LocalSparkContext {
     val numInternalAccums = TaskMetrics.empty.internalAccums.length
     // We ran 2 stages, so we should have 2 sets of internal accumulators, 1 for each stage
     assert(AccumulatorContext.numAccums === numInternalAccums * 2)
+    val cnt = 0
     val accumsRegistered = sc.cleaner match {
-      case Some(cleaner: SaveAccumContextCleaner) => cleaner.accumsRegisteredForCleanup
+      case Some(cleaner: SaveAccumContextCleaner) => {
```

Comment:
accumsRegistered is not used now. You don't need to count the number of elements, just get its size. Then you can assert for each that it isDefined rather than != None.